### ISSUE TITLE
Is it possiable using [png](https://lib.rs/crates/png) for decode emoji png?

### ISSUE THREAD

**Issue by declantsien** [dfrg/swash#48]
I am having an issue using swash with WebRender glyph rasterizer. The swash-demo works just fine. So I guess it's that webrender using different gl shader when rendering font glyphs.

Without any changes to swash this is output of emojis using [wr_glyph_rasterizer](https://github.com/declantsien/webrender/blob/master/wr_glyph_rasterizer/src/backend/swash/font.rs)

![swappy-20240305_122521](https://github.com/dfrg/swash/assets/51986034/51bfc3fc-f1fe-48bc-b0b7-3cb0b37f81ef)

[My fork](https://github.com/declantsien/swash/commit/ba2426ddbb816d8cc8fed5c97bb18b7316a85b94) using [png crate](https://lib.rs/crates/png)'s decode

![swappy-20240305_122140](https://github.com/dfrg/swash/assets/51986034/458dd698-cd19-4372-9d32-b4ccc050630f)

**Comment by notgull** [comment]
Agreed on this count. `swash` shouldn't try to re-invent the wheel.

**Comment by declantsien** [comment]
Also resizing the emoji pngs is not necessary in my case. wr_glyph_rasterizer just need to know the scale of the [Image](https://docs.rs/swash/latest/swash/scale/image/struct.Image.html)
